packages/cli/test/unit/commands/build/index.test.ts

```diff
@@ -1025,6 +1025,36 @@ describe('build', () => {
     }
   });
 
+  it('should set VERCEL_PROJECT_SETTINGS_ environment variables', async () => {
+    if (process.platform === 'win32') {
+      // this test runs a build command with `mkdir -p` which is unsupported on Windows
```

Comment:
Convert to a Node.js based build script instead?

Reply:
Done in 6b968c20bafe37cadfb7c9af0800cde53a1e8630